opencti-platform/opencti-front/src/private/components/common/cards/GenericAttackCard.tsx

```diff
@@ -155,17 +171,17 @@ GenericAttackCardProps
         >
           <CardHeader
             classes={{ root: classes.header, title: classes.title }}
-            avatar={cardData.images && cardData.images.length > 0 ? (
+            avatar={ image && image[0] ? (
               <img
                 style={{ height: '30px' }}
-                src={getFileUri(cardData.images[0].id)}
-                alt={cardData.images[0].name}
+                src={getFileUri(image[0].node.id)}
+                alt={image[0].node.name}
               />
             ) : (
               <ItemIcon type={entityType} size="large" />
             )
             }
-            title={cardData.name}
+            title={renderThreatActorIndividual(cardData)}
```

Comment:
This component is generic and used for Malware, Campaign, Intrusion Sets, ... not only Threat Actors individuals. How does it work for them ?